indexer/indexer.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/rpc"
-	sentry "github.com/getsentry/sentry-go"
+	"github.com/getsentry/sentry-go"
```

Comment:
Is sentry still used? I thought the code was deleted

Reply:
It's deleted later in the stack - I did this to make the build work.